AppInspector/MetaData.cs

```diff
@@ -206,26 +199,18 @@ public MetaData(string applicationName, string sourcePath)
 
             //Initial value for ApplicationName may be replaced if rule pattern match found later
             Matches = new List<MatchRecord>();
-
-            //initialize standard set groups using dynamic lists variables that may have more than one value
-            KeyedPropertyLists = new Dictionary<string, ConcurrentDictionary<string,byte>>
-            {
-                ["strGrpPackageTypes"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpAppTypes"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpFileTypes"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpUniqueTags"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpOutputs"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpTargets"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpOSTargets"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpFileExtensions"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpFileNames"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpCPUTargets"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpCloudTargets"] = new ConcurrentDictionary<string,byte>(),
-                ["strGrpUniqueDependencies"] = new ConcurrentDictionary<string,byte>()
-            };
-
+            PackageTypes = new List<string>();
```

Comment:
Why not declare these inline with the declaration instead of in the constructor?  Would keep the constructor cleaner.